docs/modules/admin/pages/openapi.adoc

```diff
@@ -1,5 +1,9 @@
+////
+Module Type: REFERENCE
+////
+
 = OpenAPI
 
-The spog-api server includes an OpenAPI specification for the hosted policies, under the `/openapi.json` path.
+The `spog-api` server includes an OpenAPI specification for the hosted policies, under the `/openapi.json` path.
```

Comment:
I searched for the "openapi.json" file, but I couldn't find it in Trustification's repo.  Am I missing something here?

Reply:
This refers to the HTTP path, so not in the git repository but on the running server. So for instance https://api.trustification.dev/openapi.json for the publicly hosted version.

Reply:
Ah, okay, that makes sense now. Thank you for explaining!